package.json

```diff
@@ -13,7 +13,7 @@
   },
   "peerDependencies": {
     "babel-core": "^6.0.0",
-    "webpack": "1 - 2 || 2.0.6-beta || 2.0.7-beta"
+    "webpack": "1 - 2 || 2.0.6-beta || 2.0.7-beta || 2.1.0-beta.0"
```

Comment:
if one adds e.g. `^` before `2.1.0-beta.0` it would also match versions like 2.1.0-beta.1 (which has been released now.

So this line would look like this:
 `"webpack": "1 - 2 || 2.0.6-beta || 2.0.7-beta || ^2.1.0-beta.0"`

(this site was really good for testing out npm version strings http://semver.npmjs.com/)